control-plane/connect-inject/webhook/container_init.go

```diff
@@ -255,6 +272,14 @@ func (w *MeshWebhook) containerInit(namespace corev1.Namespace, pod corev1.Pod,
 					Drop: []corev1.Capability{"ALL"},
 				},
 			}
+		} else if !w.EnableCNI && isWindows(pod) {
+			// To disable ip tables on Windows, we just need to pass an empty string as a value for this envVar.
+			redirectTrafficConfig := ""
+			container.Env = append(container.Env,
+				corev1.EnvVar{
+					Name:  "CONSUL_REDIRECT_TRAFFIC_CONFIG",
+					Value: redirectTrafficConfig,
+				})
 		}
```

Comment:
Similar to my above comment, I think we should fail here instead and give an error to the user if iptables is not supported on Windows. It should be sufficient to fail just when tproxy is enabled because CNI will only run if tproxy is true.